src/actions/admin.ts

```diff
@@ -91,12 +97,12 @@ export function AdminActions<ActionsBase extends Constructor<TinlakeParams>>(Bas
     }
 
     // ------------ admin functions lender-site -------------
-    setEquityRatio = async (amount: string) => {
-      const txHash = await executeAndRetry(this.contracts['ASSESSOR'].file, ['minJuniorRatio', amount, this.ethConfig]);
+    setMinimumJuniorRatio = async (ratio: string) => {
+      const txHash = await executeAndRetry(this.contracts['ASSESSOR'].file, [web3.fromAscii('minJuniorRatio'), ratio, this.ethConfig]);
```

Comment:
not a major thing that needs to be fixed necessarily in this PR, but maybe we can use ethers.js utils `formatBytes32String` instead of web3 to slowly phase out of web3?